package-lock.json

```diff
@@ -1,2633 +1,8 @@
 {
   "name": "ddcc-compliance-testing-framework",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Is there a reason the entire packages object is deleted?

Reply:
Good catch - my other project uses an older version of npm so I must have run `npm install` under the older version by mistake.